frontend/src/Errors.tsx

```diff
@@ -1,11 +1,18 @@
 import * as React from 'react';
+import {AppStateConsumer} from './AppStateProvider';
+import Timeout from './Timeout';
 
 type Props = {
   errors: string[];
 };
 
 const Errors = (props: Props) => (
   <div className="errors">
+    <AppStateConsumer>
+      {({updateState}) => (
+        <Timeout timeout={5000} callback={() => updateState({errors: []})} />
```

Comment:
huh. Was this really what we were doing before?

I had intended to remove only the entry we just added after 5s (i.e. a new error would not refresh the timeouts on already-displayed errors). Can probably file an issue for this and treat it as nonblocking though!

Reply:
no, this is different from what we were doing before. It was hard to get the old behavior working because
- if the responsibility is handled by the webhook handler, the handler can't be just a function
- if the responsibility is handled by the error display, then it needs to be able to hold some state from when it last rendered to prevent new errors from resetting setTimeouts